angr/analyses/reaching_definitions/rd_state.py

```diff
@@ -235,6 +237,25 @@ def _set_initialization_values(self, subject: Subject, rtoc_value: Optional[int]
 
         return self
 
+    def _generate_call_string(self, current_address: int):
```

Comment:
Nit: We can save many conversations from lists to tuples by making this function return tuples of ints instead of lists of ints.

Reply:
Sounds good to me